src/corehost/build.sh

```diff
@@ -41,14 +41,21 @@ init_rid_plat()
     if [ "$(uname -s)" == "Darwin" ]; then
         __rid_plat=osx.10.12
     fi
+    if [ "$(uname -s)" == "FreeBSD" ]; then
+        major_ver=`uname -U | cut -b1-2`
+        __rid_plat=freebsd.$major_ver
+    fi
 
     if [ $__linkPortable == 1 ]; then
         if [ "$(uname -s)" == "Darwin" ]; then
             __rid_plat="osx"
+        elif [ "$(uname -s)" == "FreeBSD" ]; then
+            __rid_plat="freebsd"
         else
             __rid_plat="linux"
         fi
     fi
+    echo __rid_plat=$__rid_plat
```

Comment:
Is this just for your testing?  or is it intended to check in?

Reply:
yes, it was for testing. removed.